lib/commands/command.bash

```diff
@@ -135,7 +135,7 @@ main() {
   plugin="$1"
   # alias="$2" -- not used directly
   version="$3"
-  install_dir="$ASDF_DATA_DIR/installs/$plugin"
+  install_dir="${asdf_data_dir)/installs/$plugin"
```

Comment:
```suggestion
  install_dir="$(asdf_data_dir)/installs/$plugin"
```

Reply:
Thank you @andrewthauer for catching that nasty typo, I have duly noted and fixed as per your commit suggestion.

Speaking about executable state of script, it seems necessary to remove executable bit, since you get two different behaviours depending on it (one of these results in asdf alias not working properly). Let me show this.

- *Case 1: command.bash is executable*

In this case, command.bash is on its own for sourcing any dependent library, such as the one providing function `asdf_data_dir`; since no loading of such library happens, when you try calling asdf alias you get:

```
❯ chmod +x <asdf-dir>/plugins/alias/lib/commands/command.bash
❯ asdf alias
<asdf-dir>/plugins/alias/lib/commands/command.bash: line 138: asdf_data_dir: command not found
```

-  *Case 2: command.bash is not executable (but it still being valid sourceable bash code)*

In this case, command.bash is sourced directly while `asdf` command is executed; all asdf libraries are available, including the one providing `asdf_data_dir` function (in fact, `<asdf-dir>/lib/utils.bash` is source here https://github.com/asdf-vm/asdf/blob/master/bin/asdf#L4 ).

So you get:

```
❯ chmod -x <asdf-dir>/plugins/alias/lib/commands/command.bash
❯ asdf alias
usage
  asdf alias <plugin> <name> [<version> | --auto | --remove]
  asdf alias <plugin> --auto
  asdf alias <plugin> [--list] 
```

I have tested this locally, both in bash and in zsh, by directly modifying my local copy of asdf-alias code.

Would you like to test it, too, in order to check whether you get the same results? Thank you